### DISCUSSION TITLE
Here are some questions about building a pregnancy model

### DISCUSSION THREAD

**Discussion by D7abc** [Open-Systems-Pharmacology/Forum#1650] [Questions & Problems]
Hello everyone, I encountered some confusion while reading the literature on pregnancy tutorials: in the final step of setting the molecular starting value, in addition to nine specific pregnancy organs, the tutorial also selected "bone" for filtration, Finally, in the "is present" section, the check box for "interstitial" was removed. May I ask if this is true for all drugs? Or how should I choose which tissue or organ to filter? How should I choose "path element2"?
![%{H$XG%KSE~$HD9DDLI_`@T](https://github.com/Open-Systems-Pharmacology/Forum/assets/141010984/bfc7aae8-b0c6-4a35-a4ee-3654097fe7c4)

I hope to receive everyone's help, thank you

**Comment by StephanSchaller** [comment]
Dear @D7abc , as a starter (don't have too much time looking into this right now), you can use our curated training material on the topic:
(https://learn-esqlabs.talentlms.com/index --> register (free for academia) and check the "pregnancy" course (updated for Version 11 of the OSP SUite).

If from Industry, you will have to go to our other portal: , e.g.: https://esqlabs.shop/products/maternal-fetal-pbpk-2

Good luck!
Stephan

**Reply by D7abc** [reply]
What is the reason why I have registered but my account is not activated

**Reply by StephanSchaller** [reply]
It depends: Did you provide your full name? And a link to a public profile for proof of your academic affiliation?
Please send inquiries to: training@esqlabs.com
